dev/source/docs/debugging-with-gdb-on-stm32.rst

```diff
@@ -28,8 +27,8 @@ on your machine following the instructions for
 :ref:`Mac <building-setup-mac>` or
 :ref:`Linux <building-setup-linux>`.
 
-A `BlackMagic probe <http://www.blacksphere.co.nz/main/index.php/blackmagic>`__ is
-also required.  They can be purchased in the US from `Transition Robotics <http://transition-robotics.com/products/black-magic-probe-mini>`__,
+A `BlackMagic probe <http://www.blacksphere.co.nz/main/index.php/blackmagic>`__ or `ST-Link V2 JST debugger <https://www.ebay.com/itm/ST-Link-V2-Stlink-Emulator-Downloader-Programming-Mini-Unit-STM8-STM32-KK/223056820813>`is
```

Comment:
`ST-Link V2 JST debugger <https://www.ebay.com/itm/ST-Link-V2-Stlink-Emulator-Downloader-Programming-Mini-Unit-STM8-STM32-KK/223056820813>`__
Otherwise it won't be link